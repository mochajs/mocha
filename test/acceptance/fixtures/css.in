body {
  font: "Helvetica Neue", Helvetica, arial, sans-serif;
  background: black;
  color: white;
}

a {
  color: blue
}
