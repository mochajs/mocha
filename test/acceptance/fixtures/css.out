body {
  font: "Helvetica Neue", Helvetica, arial, sans-serif;
  background: black;
  color: #fff;
}

a {
  color: blue;
}

foo {
  bar: 'baz';
}
